test/parallel/test-buffer-write.js

```diff
@@ -106,3 +106,17 @@ assert.strictEqual(Buffer.alloc(4)
   assert.strictEqual(buf.write('ыы', 1, 'utf16le'), 4);
   assert.deepStrictEqual([...buf], [0, 0x4b, 0x04, 0x4b, 0x04, 0, 0, 0]);
 }
+
+{
+  let i = 0;
+
+  while (i < 1_000_000) {
```

Comment:
Having a non-deterministic test is unpleasant.

Reply:
You can trigger fast api call using the method @targos implemented in his previous pull requests.